### ISSUE TITLE
Older videos show "Failed to play video"

### ISSUE THREAD

**Issue by jayrope** [Chocobozzz/PeerTube#6437]
### Describe the current behavior

Older videos, that used to play now return a "Failed to play video" error message. No explanation given.

### Steps to reproduce

1. Go to https://video.ploud.fr/w/e1rarX3brbw7jVgwAzAvVG 
2. Press Play
3. See error message

More videos, which used to play:  
https://video.ploud.fr/w/98qM5mdw4PgTJTwqyy6Lts
https://video.ploud.fr/w/wH3VCFTs3baae2eig5tHDe

All show the same behaviour. User backend shows to obvious option to reencode videos, if necessary. Replacing a failing video by a new upload is also not implemnted, it seems.

### Describe the expected behavior

Video should play.

### Additional information

* PeerTube instance:
  * URL: video.ploud.fr
  * Version: 6.10
  * NodeJS version: unknown
  * Ffmpeg version: unknown

* Browser name, version and platforms on which you could reproduce the bug: macos Ventura, Firefox 126.01 - Safari 17.2.1
* 
@Chocobozzz

**Comment by HerrIrrtum** [comment]
Same Issue on same instance. When trying to download such a video I get (made it a bit more readable):

```
type:   https://docs.joinpeertube.org/api-rest-reference.html#section/Errors/Error
detail: ENOENT: no such file or directory, stat '/var/www/peertube/versions/peertube-v6.1.0/storage/web-videos/69564f4c-649b-4943-b759-340f32f8dade-1080.mp4'
status: 404
code:   Error
error:  ENOENT: no such file or directory, stat '/var/www/peertube/versions/peertube-v6.1.0/storage/web-videos/69564f4c-649b-4943-b759-340f32f8dade-1080.mp4'
```
Maybe it is "just" an instance issue. However, the directly named part of the path "/peetube-v6.1.0/" makes me wonder if it has something to do with it, as this video was uploaded 3 years ago - there was no Peertube v. 6.1.0 back then.

**Comment by Chocobozzz** [comment]
Hi,

I think it's a configuration issue. Do you use Docker or a classic install?

**Comment by jayrope** [comment]
@Chocobozzz Neither, nor. Both @HerrIrrtum and me are long time users of video.ploud.fr 

Should be noted, that there is no user option existing to update already existing videos on a Peertube server, if a format change was introduced last year - i assume in v 6.

**Comment by Chocobozzz** [comment]
Please contact your administrator so we can investigate and fix the issue :)

**Comment by jayrope** [comment]
> Please contact your administrator so we can investigate and fix the issue :)

Thank you for the suggestion; @Chocobozzz  but video.ploud.fr offers no public way to be contacted by users other than for video-related abuse requests. 
I was under the impression, that Peertube works closely together with ploud.fr, i suppose i am wrong?

**Comment by Chocobozzz** [comment]
Hi,

We don't have the contact of video.ploud.fr admins. You can try to reach them using the email address from from https://video.ploud.fr/about/instance

**Comment by Chocobozzz** [comment]
Closing this issue due to inactivity, and I suspect a sysadmin issue.

But it highlights the fact that we need a system where admin and users can communicate in PeerTube. (related: https://github.com/Chocobozzz/PeerTube/issues/3805)

**Comment by jayrope** [comment]
Well, this was neither completed, nor solved. 
The issue remains. #3805 is only half-related, as it asks for ways to send emails from sysadmins to users - what we need here, goes vice versa: A technical contact for users with difficulties, and you're right, in general "we need a system where admin and users can communicate in PeerTube".

**Comment by Utopiah** [comment]
Hi, I'm the admin of video.benetou.fr and encountering that problem, e.g https://video.benetou.fr/w/v3aBUd7jYxuaQbWBwnDNyv

I updated from 5.x to 5.3.2 then 6.3.1, all via Docker and trying to be mindful of https://github.com/Chocobozzz/PeerTube/blob/develop/CHANGELOG.md

I updated easily, checked a video which worked right away then moved on.

Now I'm checking some videos, like the one linked above, and get that error.

Actually by browsing through my videos it seems to only after the recent ones, e.g videos 3 years ago are fine, videos that are less a year ago are not.
